app_flutter/lib/main.dart

```diff
@@ -11,6 +15,13 @@ void main() => runApp(MyApp());
 class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    if (kIsWeb) {
+      /// This application is displayed on a TV that is hard to reach.
+      /// Refreshing this page ensures the TV stays relatively up to date without
+      /// anyone having to get on a ladder to use the TV as a computer.
+      Timer.periodic(const Duration(days: 1), (_) => html.window.location.reload());
```

Comment:
I would put this in main, before runapp. @nturgut @mdebbar is there a way to refresh the page using the regular navigator controls?

Reply:
@jonahwilliams not that I know of. We could implement this as a platform message that the engine receives and refreshes the page. What would this be equivalent to on mobile? Close the app and reopen it? Is that even possible?

Reply:
No, I don't think so. I wouldn't worry about adding the functionality, just curious.

Reply:
If that's the case, is there a way to have a conditional import have `dart:html` only on the Web platform.